torch/csrc/jit/ir/ir.h

```diff
@@ -423,6 +423,7 @@ struct TORCH_API Node {
     return scope_->namesFromRoot();
   }
 
+  // Copies the source range, scope and callstack from another node.
```

Comment:
@kit1980 I added a comment here outside of onnx merge rules, please help review this PR.